cmd/cli/main.go

```diff
@@ -2,39 +2,47 @@ package main
 
 import (
 	"flag"
-	"strings"
+	"log/slog"
+	"os"
 
 	"github.com/flashbots/go-utils/envflag"
-	"github.com/flashbots/go-utils/logutils"
-	"go.uber.org/zap"
 )
 
 var (
 	version = "dev" // is set during build process
 
-	logDev     = envflag.MustBool("log-dev", false, "log in development mode (plain text)")
-	logLevel   = envflag.String("log-level", "info", "log level (one of: \""+strings.Join(logutils.Levels, "\", \"")+"\")")
-	logService = envflag.String("log-service", "your-project", "\"service\" tag to logs")
+	logProd    = envflag.MustBool("log-prod", false, "log in production mode (json)")
+	logDebug   = envflag.MustBool("log-debug", false, "log debug messages")
+	logService = envflag.String("log-service", "", "'service' tag to logs")
+
+	log *slog.Logger
```

Comment:
**question (non-blocking):**

isn't the use of global vars frowned upon?  I am not that big of an expert in go, asking just for own sake.

Reply:
it's not great, but makes it a lot easier to use in quick scripts instead of passing the pointer around. definitely not best practices and probably should get removed 👍